fizzbuzz_yamaguchi.py

```diff
@@ -0,0 +1,9 @@
+for i in range(1, 101):
```

Comment:
Some comment on what's happening would be helpful for others to use the code.
E.g.: # Print fizzbuzz (https://en.wikipedia.org/wiki/Fizz_buzz) for numbers 1 to 100.